rust/hyperlane-base/build.rs

```diff
@@ -0,0 +1,8 @@
+use anyhow::Result;
+use vergen::EmitBuilder;
+
+fn main() -> Result<()> {
+    EmitBuilder::builder().all_git().emit()?;
```

Comment:
nit: Looks like `EmitBuilder::builder().all_git().git_sha()` would suffice here, to avoid adding unnecessary env vars

Reply:
It makes sense. 

I fixed it like this: 
```suggestion
    EmitBuilder::builder().git_sha(false).emit()?;
```